### PR TITLE
Update lxml dependency to the latest security patch

### PR DESCRIPTION
**Pull request by Elkasitu**
Version 4.9.1 addresses a NULL pointer dereference vulnerability

### CODE REVIEW

**Review by osoukup — CHANGES_REQUESTED** [review]
It seems that we also need this
https://docs.djangoproject.com/en/4.0/releases/3.2.14/
to get pipelines green

**Comment by Elkasitu** [comment]
@osoukup it's done in #4

**Comment by osoukup** [comment]
I probably did the review in a wrong order then. LGTM

File: devel-requirements.in [hunk 1]
```diff
@@ -6,7 +6,6 @@ djangorestframework-stubs
 factory-boy
 freezegun
 k5test
-lxml
```
Comment (JakubFrejlach) [inline devel-requirements.in:9]: Just an idea, we do have a `testenv:mypy` in our `tox.ini` and according to [this](https://github.com/pre-commit/mirrors-mypy/issues/30), the `lxml` is probably needed to run mypy with the `--html-report` parameter. :thinking:

Comment (Elkasitu) [reply devel-requirements.in:9]: Good catch, I will just update the lib in that case

**Pull request merged**